src/YaccConstructor/Program.fs

```diff
@@ -88,37 +118,64 @@ let () =
             Some tmpName
         else None
 
+    let generatorNameD = 
+            match !generatorName with
+            |Some x->x
+            |None->""
+
     let generateSomething = ref true
 
     let printItems iName items deft =
-        fun _ ->
             generateSomething := false
             printfn "\nAvailable %s: " iName
             Seq.map (fun x -> x + (if Some x = deft then " (default)" else "")) items
             |> String.concat "\n    "
             |> fun x -> printf "    %s\n" x
+    
+        
+    let argv = System.Environment.GetCommandLineArgs()
+    let parser = ArgumentParser.Create<CLIArguments>()
+    let args = parser.Parse argv.[1..]
+
 
-    let commandLineSpecs =
-        ["-f", ArgType.String (fun s -> feName := Some s), "Frontend name. Use -af to list available."
-         "-af", ArgType.Unit (printItems "frontends" addinFrontendNames !feName), "Available frontends"
-         "-g", ArgType.String 
-            (fun s -> 
-                match Array.toList (s.Split ' ') with
+    feName := args.GetResult (<@ F @>, defaultValue = !feName)
+    
+
+    let genString= args.GetResult (<@ G @>, defaultValue = generatorNameD)
+    let gen = 
+        match Array.toList (genString.Split ' ') with
```

Comment:
Is it possibe to use [SubComands](https://fsprojects.github.io/Argu/tutorial.html#SubCommands) for generator options?